### ISSUE TITLE
mmark 0.0.2 causes test failure for mmark-ext 0.0.1.0

### ISSUE THREAD

**Issue by DanBurton** [mrkkrp/mmark-ext#5]
Ping @mrkkrp 

```
  tests/Text/MMark/Extension/CommonSpec.hs:101:
  1) Text.MMark.Extension.Common.toc works
```

The expected/actual diff is huge so not included here. When I repro'd locally, though, I did notice that the "actual" section had some `\n` characters which the "expected" section did not.

```html
<!-- expected -->
...
<ul>\n<li><a href=\"#nullam-luctus\">Nullam luctus</a><ul>\n<li><a href=\"#sed-euismod\">Sed euismod</a><ul>\n<li><a href=\"#curabitur-ullamcorper\">Curabitur ullamcorper</a><ul>\n<li><a href=\"#ut-dictum-quis\">Ut dictum quis</a><ul>\n<li><a href=\"#mauris-efficitur\">Mauris efficitur</a></li>\n</ul>\n</li>\n</ul>\n</li>\n</ul>\n</li>\n</ul>\n</li>\n<li><a href=\"#aenean-auctor-nec\">Aenean auctor nec</a></li>\n</ul>
...
<!-- actual -->
...
<ul>\n<li>\n<a href=\"#nullam-luctus\">Nullam luctus</a>\n<ul>\n<li>\n<a href=\"#sed-euismod\">Sed euismod</a>\n<ul>\n<li>\n<a href=\"#curabitur-ullamcorper\">Curabitur ullamcorper</a>\n<ul>\n<li>\n<a href=\"#ut-dictum-quis\">Ut dictum quis</a>\n<ul>\n<li>\n<a href=\"#mauris-efficitur\">Mauris efficitur</a>\n</li>\n</ul>\n</li>\n</ul>\n</li>\n</ul>\n</li>\n</ul>\n</li>\n<li>\n<a href=\"#aenean-auctor-nec\">Aenean auctor nec</a>\n</li>\n</ul>
...
```

Here's how I reproduced locally:

```bash
$ stack unpack mmark-ext-0.0.1.0 && cd mmark-ext-0.0.1.0
$ vi stack.yaml # added the following stack.yaml
$ stack test
```

```yaml
# stack.yaml
resolver: nightly-2017-12-07
extra-deps:
- mmark-0.0.2.0
```

**Comment by mrkkrp** [comment]
Ah, my bad, forgot to update `mmark-ext`. I'll do it now.

**Comment by mrkkrp** [comment]
Fixed, `mmark-ext-0.0.1.1` is uploaded on Hackage.
